vendor/google.golang.org/api/compute/v0.beta/compute-gen.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 Google LLC.
+// Copyright 2022 Google LLC.
```

Comment:
This change is very suspicious. There should be no changes in vendor/google.golang.org/api/compute/v0.beta/compute-gen.go

Reply:
I believe the client files were manually updated in https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1101

@sunnylovestiramisu @mattcary I believe these were custom-generated sdk files right? I think that means we should manually revert the change in this PR.

Reply:
We manually generated the beta client intentionally to have the iops field.